ui/src/pages/Alert/Subscriptions/SubscripDrawerForm.tsx

```diff
@@ -102,6 +104,7 @@ export default function SubscripDrawerForm({
                 labelFormName="name"
                 valueFormName="value"
                 regBoxFormName="isRegex"
+                form={form}
```

Comment:
This is not a good pattern. `Form.Item` is controlled by `Form`, you added dependency from item to form,  which construct a control cycle here.

Reply:
If you want to set field values in `form`, define a controlled form item instead. Reference of controlled form item could be found at https://ant-design.antgroup.com/components/form-cn#components-form-demo-customized-form-controls

Reply:
Review again?